src/doc/unstable-book/src/language-features/rustc-private.md

```diff
@@ -12,3 +12,26 @@ The presence of this feature changes the way the linkage format for dylibs is ca
 that is necessary for linking against dylibs that statically link `std` (such as `rustc_driver`).
 This makes this feature "viral" in linkage; its use in a given crate makes its use required in
 dependent crates which link to it (including integration tests, which are built as separate crates).
+
+
+
+### Required Components
+
+To use the `rustc_private` feature, you need to install:
+
+1. The `rustc-dev` component (provides compiler libraries)
+2. The `llvm-tools` component (provides LLVM libraries needed for linking)
+
+You can install these components using `rustup`:
+
+```
+rustup component add rustc-dev llvm-tools
+```
+
+Without the `llvm-tools` component, you may encounter linking errors like:
+
+```
+error: linking with `cc` failed: exit status: 1
+  |
+  = note: rust-lld: error: unable to find library -lLLVM-{version}
+```
```

Comment:
This is *only* the case if you use an official (or offical-like) toolchain that goes through `rustup` (for instance, official toolchain distributions). However, you can build custom toolchains yourself where this is *not* the solution (you may have to explicitly configure library search paths for a custom LLVM). You may *not* be using rustup or can't use rustup in the first place.

I would maybe rebrand this as something like "`rustc_private` and official toolchains shipped via rustup", or maybe not mention this in the error docs at all.